packages/tables/src/Columns/Text.php

```diff
@@ -85,4 +86,15 @@ public function options($options)
 
         return $this;
     }
+
+    public function limit($length = -1)
+    {
+        $this->formatUsing = function ($value) use ($length) {
+            $value = Str::of($value)->limit($length);
```

Comment:
No need for the `Str::of()` here - you could just use `Str::limit` directly, right?

Reply:
You can indeed! I didn't realise you could do that, buzzing I've learned something new this morning :)

Reply:
I've updated this to:

```php
return Str::limit($value, $length);
```

Reply:
Nice one @jamessessford!